### ISSUE TITLE
Messages failing to send to particular user

### ISSUE THREAD

**Issue by goosekrk** [signalapp/Signal-Android#4460]
I finally got my girlfriend to use Signal on Android, and it's been great, up until last night she suddenly stopped being able to send messages to a non-Signal friend, but could still text me. She uninstalled and was able to use her old SMS app without issue.

Tonight, I was texting with a non-Signal friend, and suddenly my text failed to send. I get the exclamation mark and the "Some issues need your attention." Rebooting, waiting, resending, using wi-fi or 4G, I cannot get this message to go. It's just plain text, short, nothing fancy, same as all the previous texts that did work.

I can still send and receive normally to other phones, SIgnal or otherwise. It's only this one person it refuses to send to, just out of the blue.

Debug log: https://gist.github.com/da13fa1cef925d9d6c99

**Comment by moxie0** [comment]
When you open the conversation with this non-signal friend, does it display "send unsecured sms" or "send signal message" in the compose field?

**Comment by goosekrk** [comment]
"Send unsecured SMS"

**Comment by ohthehugemanatee** [comment]
I've encountered this problem too, with both Signal and non-Signal friends. Today I got it with a signal friend, with whom I've texted many times. He had recently regenerated his key, so I had to restart the encrypted tunnel. That worked fine, but messages still failed with the un-helpful "failed to send" message. I plugged in USB to fire up logcat... and the message went through just fine. 

This suggests to me some kind of temporary service outage with google, rather than anything fundamental to Signal itself. Perhaps the best solution is clearer error messages for "message failed" conditions?
